.github/workflows/test_coverage.yml

```diff
@@ -37,11 +37,11 @@ jobs:
         run: |
           python -m pip install build
           python -m build
-          find . -name "*.whl" -exec python -m pip install {}[test_full] ';'
+          find . -name "*.whl" -exec python -m pip install {}[devel_full] ';'
 
       - name: clean environment
         run: |
-          export backup_files=( tests tutorials .env_file makefile .coveragerc )
+          export backup_files=( tests tutorials .env_file makefile .coveragerc pytest.ini docs )
```

Comment:
We need `docs` for `test_coverage` now?..

Reply:
Yes.
`tutorial_dependencies` tests require `docs/source/utils/notebook.py` (although only to get regex pattern from `InstallationCell`).

Reply:
Well, maybe we should copy the required files only in that case?